python/ray/util/placement_group.py

```diff
@@ -109,11 +114,17 @@ def bundle_count(self):
         return len(self.bundle_cache)
 
     def _get_none_zero_resource(self, bundle: List[Dict]):
-        # Any number between 0-1 should be fine.
-        INFEASIBLE_TASK_SUPPRESS_MAGIC_NUMBER = 0.0101
+        # Set a mock value to schedule a dummy task.
+        MOCK_VALUE = 0.001
         for key, value in bundle.items():
             if value > 0:
-                value = INFEASIBLE_TASK_SUPPRESS_MAGIC_NUMBER
+                if key == "memory":
+                    # Make sure the memory resource can be
+                    # transformed to memory unit.
+                    to_memory_units(value, True)
+                    value = MEMORY_RESOURCE_UNIT_BYTES
```

Comment:
@rkooo567  I use the constant `MEMORY_RESOURCE_UNIT_BYTES` for memory resource here because this value can be changed in the future( I remember the original value is 50M )